README.md

```diff
@@ -1 +1,14 @@
-# data_generator_app
\ No newline at end of file
+# Data Generator App
+
+Application generates test data in JSON fromat in a specified schema.
+
+
+## Usage: <br />
+1. python3 -m pip install .
+2. data_generator [-h] [--file_count FILE_COUNT] [--file_name FILE_NAME] [--file_prefix {count,random,uuid}] [--data_schema DATA_SCHEMA] 
```

Comment:
After installing you will face with a problem that Python doesn't know what is data_generator. Yes, it works in your environment, but will not work in others. 
To fix that you need to specify PYTHONPATH variable with project directory before executing data_generator. For example:
`> python3 -m pip install .`
`...success`
`> echo $PYTHONPATH=PATH_TO_YOUR_PROJECT`
`> data_generator ...`

So, please add this tip to your README. You can try to run your app from different python env to see what happens:)